README.md

```diff
@@ -928,6 +928,23 @@ Other Style Guides
     );
     ```
 
+  <a name="functions--prefer-rest-params"></a>
+  - [7.16](#functions--prefer-rest-params) Prefer the use `...rest` operator instead of `arguments` keyword for taking n number of arguments in function. eslint: [`prefer-rest-params`](https://eslint.org/docs/rules/prefer-rest-params)
```

Comment:
rest syntax isn't an operator
```suggestion
  - [7.16](#functions--prefer-rest-params) Prefer the use `...rest` syntax instead of the `arguments` keyword for taking a variable number of arguments in a function. eslint: [`prefer-rest-params`](https://eslint.org/docs/rules/prefer-rest-params)
```

Reply:
done